src/nvim/os_unix_defs.h

```diff
@@ -1,216 +0,0 @@
-# define INDENT_FILE    "indent.vim"
-#endif
-#ifndef FTOFF_FILE
-# define FTOFF_FILE     "ftoff.vim"
-#endif
-#ifndef FTPLUGOF_FILE
-# define FTPLUGOF_FILE  "ftplugof.vim"
-#endif
-#ifndef INDOFF_FILE
-# define INDOFF_FILE    "indoff.vim"
-#endif
-#ifndef SYS_MENU_FILE
-# define SYS_MENU_FILE  "$VIMRUNTIME/menu.vim"
-#endif
-
-#ifndef USR_EXRC_FILE
-#  define USR_EXRC_FILE "~/.exrc"
-#endif
-
-
-#ifndef USR_VIMRC_FILE
-#  define USR_VIMRC_FILE "~/.nvimrc"
-#endif
-
-
-#if !defined(USR_EXRC_FILE2)
-#    define USR_VIMRC_FILE2     "~/.nvim/nvimrc"
-#endif
-
-# ifndef VIMINFO_FILE
-#   define VIMINFO_FILE "~/.nviminfo"
-# endif
-
-#ifndef EXRC_FILE
-# define EXRC_FILE      ".exrc"
-#endif
-
-#ifndef VIMRC_FILE
-# define VIMRC_FILE     ".nvimrc"
-#endif
-
-
-#ifndef SYNTAX_FNAME
-# define SYNTAX_FNAME   "$VIMRUNTIME/syntax/%s.vim"
-#endif
-
-#ifndef DFLT_BDIR
-#   define DFLT_BDIR    ".,~/tmp,~/"    /* default for 'backupdir' */
-#endif
-
-#ifndef DFLT_DIR
-#   define DFLT_DIR     ".,~/tmp,/var/tmp,/tmp" /* default for 'directory' */
-#endif
-
-#ifndef DFLT_VDIR
-#   define DFLT_VDIR    "~/.nvim/view"       // default for 'viewdir'
-#endif
-
-#define DFLT_ERRORFILE          "errors.err"
-
-#  ifdef RUNTIME_GLOBAL
-#   define DFLT_RUNTIMEPATH     "~/.nvim," RUNTIME_GLOBAL ",$VIMRUNTIME," \
-  RUNTIME_GLOBAL "/after,~/.nvim/after"
-#  else
-#   define DFLT_RUNTIMEPATH \
-  "~/.nvim,$VIM/vimfiles,$VIMRUNTIME,$VIM/vimfiles/after,~/.nvim/after"
-#  endif
-
-/* Special wildcards that need to be handled by the shell */
-#define SPECIAL_WILDCHAR    "`'{"
-
-/*
- * Unix has plenty of memory, use large buffers
- */
-#define CMDBUFFSIZE 1024        /* size of the command processing buffer */
-
-/* Use the system path length if it makes sense. */
-#if defined(PATH_MAX) && (PATH_MAX > 1000)
-# define MAXPATHL       PATH_MAX
-#else
-# define MAXPATHL       1024
-#endif
-
-# ifndef DFLT_MAXMEM
-#  define DFLT_MAXMEM   (5*1024)         /* use up to 5 Mbyte for a buffer */
-# endif
-# ifndef DFLT_MAXMEMTOT
-#  define DFLT_MAXMEMTOT        (10*1024)    /* use up to 10 Mbyte for Vim */
-# endif
-
-#if !defined(S_ISDIR) && defined(S_IFDIR)
-# define        S_ISDIR(m) (((m) & S_IFMT) == S_IFDIR)
-#endif
-#if !defined(S_ISREG) && defined(S_IFREG)
-# define        S_ISREG(m) (((m) & S_IFMT) == S_IFREG)
-#endif
-#if !defined(S_ISBLK) && defined(S_IFBLK)
-# define        S_ISBLK(m) (((m) & S_IFMT) == S_IFBLK)
-#endif
-#if !defined(S_ISSOCK) && defined(S_IFSOCK)
-# define        S_ISSOCK(m) (((m) & S_IFMT) == S_IFSOCK)
-#endif
-#if !defined(S_ISFIFO) && defined(S_IFIFO)
-# define        S_ISFIFO(m) (((m) & S_IFMT) == S_IFIFO)
-#endif
-#if !defined(S_ISCHR) && defined(S_IFCHR)
-# define        S_ISCHR(m) (((m) & S_IFMT) == S_IFCHR)
-#endif
-
-/* Note: Some systems need both string.h and strings.h (Savage).  However,
- * some systems can't handle both, only use string.h in that case. */
-# include <string.h>
-#if defined(HAVE_STRINGS_H) && !defined(NO_STRINGS_WITH_STRING_H)
```

Comment:
> In Apple's strings.h, string.h is included and has include guards, so I don't see the problem. Maybe it was for an older version of macos ?

I think that might be the case. Assuming it passes on Travis then I'd imagine it's fine (it shouldn't even compile otherwise).